pyaedt/modules/Material.py

```diff
@@ -396,8 +378,8 @@ def add_thermal_modifier_dataset(self, dataset_name, index=0):
 
         Returns
         -------
-        bool
-            ``True`` when successful, ``False`` when failed.
+        type
```

Comment:
```suggestion
        bool
            ``True`` when successful, ``False`` when failed.
```
Same reasoning as above.